test/clj/game/core_test.clj

```diff
@@ -548,17 +548,20 @@
    (get-in @state [:runner :play-area pos])))
 
 (defn rez-impl
-  [state card & args]
-  (let [card (get-card state card)]
-    (is' (installed? card) (str (:title card) " is installed"))
-    (is' (not (rezzed? card)) (str (:title card) " is unrezzed"))
-    (when (and (installed? card)
-               (not (rezzed? card)))
-      (core/process-action "rez" state :corp (merge {:card card} (first args))))))
+  ([state card] (rez-impl state card nil))
+  ([state card {:keys [expect-no-rez]}]
+   (let [card (get-card state card)]
+     (is' (installed? card) (str (:title card) " is installed"))
+     (is' (not (rezzed? card)) (str (:title card) " is unrezzed"))
+     (when (and (installed? card)
+                (not (rezzed? card)))
+       (core/process-action "rez" state :corp {:card card})
+       (if expect-no-rez (is' (not (rezzed? (get-card state card))) (str (:title card) " is still unrezzed"))
+                         (is' (rezzed? (get-card state card)) (str (:title card) " is rezzed")))))))
```

Comment:
I prefer to put both branches on their own lines.

Reply:
👍🏻